tools/depends/target/ffmpeg/FFMPEG-VERSION

```diff
@@ -1,5 +1,5 @@
 LIBNAME=ffmpeg
 BASE_URL=https://github.com/xbmc/FFmpeg/archive
-VERSION=3.1.6-Krypton
+VERSION=release/3.3-kodi
```

Comment:
You need all of them, otherwise cmake won't pick up external ffmpeg and fall back to internal build
```
set(_avcodec_ver ">=57.87.100")
set(_avfilter_ver ">=6.80.100")
set(_avformat_ver ">=57.69.100")
set(_avutil_ver ">=55.55.100")
set(_swscale_ver ">=4.4.100")
set(_swresample_ver ">=2.5.100")
set(_postproc_ver ">=54.3.100")

```